### DISCUSSION TITLE
How to forcefully change RustDesk client ID

### DISCUSSION THREAD

**Discussion by alexchini** [rustdesk/rustdesk#6197] [General]
### Bug Description

I have a set of computers with **Ubuntu 22.04** generated from a clone image. Naturally, all the computers have the same RustDesk ID, so I would like to change it, but in the latest versions, it doesn't seem possible.

Following an issue from last year, I tried to change the machine-id and delete the file /.config/rustdesk/RustDesk.toml, but after restarting the computer and RustDesk, even though the machine-id has changed, the RustDesk ID remains the same.

I'm running with my own rustdesk server.

### How to Reproduce

as describe before

### Expected Behavior

change client id

### Operating system(s) on local side and remote side

Windows 11 -> Ubuntu 22.04.1 LTS

### RustDesk Version(s) on local side and remote side

1.2.3 --> 1.2.3

**Comment by alexchini** [comment]
Ok seems i ve found a possible solution, it need to remove all config files RustDesk.toml also that of root.

**Reply by rijotech** [reply]
I did that on Ubuntu 24.04, and the ID didn't change
